cmd/merge/main.go

```diff
@@ -97,13 +97,18 @@ func main() {
 	fmt.Println(string(out))
 }
 
-func unmarshal(path string) (*org.Config, error) {
+func unmarshalFromFile(path string) (*org.Config, error) {
 	buf, err := ioutil.ReadFile(path)
 	if err != nil {
 		return nil, fmt.Errorf("read: %v", err)
 	}
+
+	return unmarshal(buf)
+}
+
+func unmarshal(buf []byte) (*org.Config, error) {
 	var cfg org.Config
-	if err := yaml.Unmarshal(buf, &cfg); err != nil {
+	if err := yaml.Unmarshal(buf, &cfg, yaml.DisallowUnknownFields); err != nil {
```

Comment:
Just read about the [DisallowUnknownFields](https://github.com/kubernetes-sigs/yaml/blob/65d71bb2ae7234d12629f43f77e1a44512057bba/yaml_go110.go#L26) . Yes, I think it's doing exactly what we needed (i.e., to flag unknown fields, rather than dropping them altogether) :+1: 

Tested it. Works beautifully!

```
FATA[0000] Failed to load orgs: merge teams config/kubernetes/org.yaml: error in config/kubernetes/sig-cli/teams.yaml: unmarshal: error unmarshaling JSON: while decoding JSON: json: unknown field "repo" 
```